app/views/rewards/index.html.erb

```diff
@@ -13,3 +13,23 @@
        <br>
       <% end %>
     </div>
+  <br>
+    <div class="pagination">
```

Comment:
I would place this whole div in a partial

Reply:
ok, fixed